centreon/packages/ui/src/Dialog/index.tsx

```diff
@@ -93,14 +93,20 @@ const Dialog = ({
       )}
       <DialogActions className={dialogActionsClassName}>
         {onCancel && (
-          <Button color="primary" disabled={cancelDisabled} onClick={onCancel}>
+          <Button
+            color="primary"
+            data-testid={labelConfirm || 'cancel'}
```

Comment:
```suggestion
            data-testid={'cancel'}
```
wrong label & test id should preferably be a constant